JobRecommendationService.md

```diff
@@ -0,0 +1,173 @@
+# Job Recommendation service
+# Contents
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+  - [JobData](#jobdata)
+    - [Job](#job)
+  - [UserData](#userdata)
+  - [UserPreferences](#userpreferences)
+    - [PreferredLocation](#preferredlocation)
+    - [Compensation](#compensation)
+    - [CaroteneTitle](#carotenetitle)
+    - [CaroteneVersion](#caroteneversion)
+- [Response Structure](#response-structure)
+- [Versioning](#versioning)
+
+----------------------------
+# Summary
+
+Job Recommendation Service is part of the profile recommendations, which recommends similar and related jobs to a user based on the jobs he applied and user preferences. The recommendations are combined from FAISS indexes and GBR data, then filtered by the applied jobs.
+
+The endpoint for Job Recommendation Service is at https://api.careerbuilder.com/core/recommendations/job
+
+----------------------------
+# Request structure
+The service supports GET and POST requests to:
+https://api.careerbuilder.com/core/search/candidatereranker.
+
+OAUTH credentials are **required**.
+
+A request is composed of 3 main parts:
+`job_data`, `user_data`, `user_preferences`.
+
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| `job_data` | [**JobData**](#jobdata) | **TRUE** | **JobData** contains a list of jobs which is needed for recommendations.
+| `user_data` | [**UserData**](#userdata) | **FALSE** | **UserData** contains visitor id and email address of the user, which is then used to retrieve applied jobs for filtering.
```

Comment:
I think this would be slightly better worded as:

> **UserData** contains the visitor id and email address for a given user. This data is used for filtering jobs that the user has already applied to from the final recommendations set.